app/js/main.js

```diff
@@ -18,6 +18,10 @@ import {DemoPaywallController} from './demo-controller';
 import {MeteringDemo} from './metering';
 import {log} from './log';
 
+const OWNS_PUBLISHER_SUBSCRIPTION = false;
```

Comment:
Hardcoding these state variables is a good start, but I think parsing them from the URL would be better for testing

Reply:
is there anything existing to pull from?

Reply:
You could use the `getQueryParams` method defined in this file

Reply:
I was more wondering if anything did something similar already and there were query parameters to reuse.

Reply:
You could add new flows in this file, instead of just adding to the "metering" flow. Each flow is triggered by the URL. Ex: "https://scenic-2017.appspot.com/1?swgButton" or "https://scenic-2017.appspot.com/1?metering"